src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -441,9 +441,13 @@ export class PatientViewPageStore {
             this.uncalledMutationData
         ],
         invoke: async() => {
-            if (this.civicGenes.status == "complete") {

```

Comment:
I think `await` waits for the status to be `complete`, so instead changed this to check if `civicGenes.result` is defined.